bin/GameIndex.dbf

```diff
@@ -6476,6 +6536,7 @@ Region = PAL-E
 Serial = SLED-53953
 Name   = FIFA Street 2 [Demo]
 Region = PAL-E
+GIFFIFOHack = 1 // Fix flag corruptions
```

Comment:
Did you actually test Fifa Street 2? I belive the flags corruption can be fixed with blending half, the other half a shader (8bit texture conversion) fixes that in OpenGL.

Reply:
Ok, can you upload a gs dump ? I want to be sure before this is merged.

Reply:
Actually better a block dump with a save state.